automated-comments/scala/general/do_not_write_to_console.md

```diff
@@ -0,0 +1,6 @@
+Try removing the functions(s) that write to the console, as the tests don't require them and they make the code slightly harder to read.
```

Comment:
Is this comment ground for a disapproval, or can it still be approved? If it is the former, maybe the "Try" should be removed?